pkg/csi_driver/gcfs_driver.go

```diff
@@ -224,5 +225,9 @@ func (driver *GCFSDriver) Run(endpoint string) {
 	//Start the nonblocking GRPC
 	s := NewNonBlockingGRPCServer()
 	s.Start(endpoint, driver.ids, driver.cs, driver.ns)
+	if driver.config.FeatureOptions.FeatureLockRelease {
```

Comment:
if feature enabled and node server is enabled, only then we should start the lock release controller

Reply:
done